src/main/main-controller.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+    'use strict';
+
+    class MainCtrl {
+      constructor($timeout, currentWindowService) {
+          this.$timeout = $timeout;
+          this.currentWindowService = currentWindowService;
+          currentWindowService.ready(this.onReady.bind(this));
+      }
+      onReady() {
+          this.window = this.currentWindowService.getCurrentWindow();
+      }
+
+      isSummarised() {
```

Comment:
I prefer compact, as that has more connotation with the window being smaller. Summary, for example, could imply the window's the same size, just showing a 'summary' of all favourites